docs/build/getting-started/safe.md

```diff
@@ -0,0 +1,72 @@
+---
+description: 'Multisig wallet refers to having multiple private keys for a single web3 wallet.'
+image: /img/logo/WASP_logo_dark.png
+---
+
+# MultiSig Wallet
+:::note TL;DR;
+Safe Wallet is IOTA's exclusive [MultiSig](https://safe.iotaledger.net/welcome) wallet. The term “
```

Comment:
Why exclusive btw? Doesn't that mean it only is available on IOTA? 😅

Reply:
Exclusivity in regards to "safe" on IOTA.
Yes, this safe is deployed on other chains also.

Reply:
Got it